### ISSUE TITLE
4.1.7 bug fixes

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1325]
In GitLab by @dbeltrankyl on Jun 12, 2024, 14:27

Fyi: @kinow 

This issue is to keep track of the issues of the 4.1.7 version.  !447 

* [X] Logs not being retrieved 

Once all branches were merged in the master, I forgot to check the logs, or it was working for me because of the following:

The issue was that the migrate added the as_misc data. Initially, I thought it was a matter of changing the experiment_data to look into the misc_data for the log recovery. However, while working on #1323, I realized that the as_misc.yml of the new experiment/ old experiments didn't have the AS_MISC parameter that I added in the migrate experiment that I was using.

So, In summary, I've uploaded 1.0.67 asconfparser to add AS_MISC and the AS_COMMAND to the as_misc.yml, and now all misc_data related code is looking at the right dictionary. 

* [X] X11 routines

I noticed that the AS was calling to X11 subroutines when it shouldn't changed the X11 get to return False/True and normalized the routines to if job.X11

The rest seems OK according to Leo testing suite https://earth.bsc.es/gitlab/digital-twins/de_340-2/workflow/-/issues/419#note_286852.
